extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -93,112 +110,409 @@ public function addPage( $id, $data ) {
+								$insert = DumpUtils::createInsert(
+									'import_revision',
+									self::COLUMNS_REVISION,
+									[
+										"revision_id" => $row->rev_id,
+										"page_id" => $row->rev_page,
+										"page_namespace" => $curPage['namespace'],
+										"title" => $title,
+										"user_type" => $this->getContributorType( $row ),
+										"user_identifier" => $row->rev_user,
+										"timestamp" => $row->rev_timestamp,
+										"is_minor_edit" => $row->rev_minor_edit,
+										"is_deleted" => $row->rev_deleted,
+										"length" => $row->rev_len,
+										"parent_id" => $row->rev_parent_id,
+										"text_flags" => $row->old_flags,
+										"raw_content" => $plainText,
+										"content" => $parsedText,
+									]
+								) . "\n";
+
+								$inserts[] = $insert;
+
+								if ( !$this->bulk ) {
+									fwrite( $fh, $insert) ;
+									fflush( $fh );
+								}
+							}
+
+							$dbh->freeResult( $result );
+						},
+						false, false
+					);
+
+				$dbh->closeConnection();
+				wfGetLB( false )->closeConnection( $dbh );
+
+				if ( $this->bulk && !empty( $inserts )) {
+					$chunks = array_chunk( $inserts, 10 );
+					foreach ( $chunks as $chunk ) {
+						$multiInsert = DumpUtils::createMultiInsert('import_revision',
+								self::COLUMNS_REVISION, $chunk) . "\n";
+						fwrite( $fh, $multiInsert );
+						fflush( $fh );
+						unset( $multiInsert );
+					}
+				}
+
+				unset( $inserts );
+
+				if ( $queryResult === false && $tries > 0 ) {
+					WikiaLogger::instance()->info( "Retry used! (rev batch load) - ".( $tries - 1 )." left" );
+				}
+
+			} while ($queryResult === false && $tries-- > 0);
+
+		}
+	}
+
+	/**
+	 * Wall_related_pages
+	 * +-------------+------------------+------+-----+-------------------+-----------------------------+
+	 * | Field       | Type             | Null | Key | Default           | Extra                       |
+	 * +-------------+------------------+------+-----+-------------------+-----------------------------+
+	 * | comment_id  | int(10) unsigned | NO   | PRI | NULL              |                             |
+	 * | page_id     | int(10) unsigned | NO   | PRI | NULL              |                             |
+	 * | order_index | int(10) unsigned | NO   |     | NULL              |                             |
+	 * | add_at      | timestamp        | NO   |     | CURRENT_TIMESTAMP | on update CURRENT_TIMESTAMP |
+	 * | last_update | timestamp        | YES  |     | NULL              |                             |
+	 * +-------------+------------------+------+-----+-------------------+-----------------------------+
+	 */
+	public function getTopics( $fh ) {
+
 		$pageIds = array_keys( $this->getPages() );
+		$pageIdsChunks = array_chunk($pageIds, 500);
+		$topicsNumber = 0;
+
+		foreach ($pageIdsChunks as $part) {
+			$dbh = DumpUtils::getDBSafe( DB_SLAVE );
+			$dbh->ping();
+			$inserts = [];
+			$queryResult = ( new \WikiaSQL() )->SELECT( "wall_related_pages.*" )
```

Comment:
after brief look I can see that only comment_id and page_id fields are used ;)

Reply:
True!